static/styles/night_mode.scss

```diff
@@ -236,6 +236,17 @@ on a dark background, and don't change the dark labels dark either. */
         color: hsl(0, 0%, 100%);
     }
 
+    #streamlist-toggle,
+    #userlist-toggle {
+        border-color: hsla(199, 33%, 46%, 0.2);
+    }
+
+    #streamlist-toggle-button,
+    #userlist-toggle-button {
+        color: #dddeee;
```

Comment:
Can you use HSL instead of hex? Also, you might as well use `color: inherit` and move the border-color declarations to the toggle buttons themselves instead of the parent elements so you don't have to change the border color in a separate selector.